Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 03:17:53
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 03:18:00] "GET / HTTP/1.1" 200 15048
[11/Oct/2023 03:18:01] "GET / HTTP/1.1" 200 15048
[11/Oct/2023 03:18:01] "GET / HTTP/1.1" 200 15048
[11/Oct/2023 03:23:43] "GET / HTTP/1.1" 200 15048
[11/Oct/2023 03:27:29] "GET / HTTP/1.1" 200 14454
[11/Oct/2023 03:27:34] "GET / HTTP/1.1" 200 14454
[11/Oct/2023 03:27:37] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 03:27:37] "GET / HTTP/1.1" 200 14733
[11/Oct/2023 03:27:38] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:27:38] "GET / HTTP/1.1" 200 14897
[11/Oct/2023 03:27:39] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:27:40] "GET / HTTP/1.1" 200 14897
[11/Oct/2023 03:27:41] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:27:41] "GET / HTTP/1.1" 200 14897
[11/Oct/2023 03:27:58] "GET / HTTP/1.1" 200 14454
[11/Oct/2023 03:27:58] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 03:27:58] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 03:27:58] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 03:28:00] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:28:00] "GET / HTTP/1.1" 200 14728
[11/Oct/2023 03:28:01] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:28:01] "GET / HTTP/1.1" 200 14728
[11/Oct/2023 03:28:02] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:28:02] "GET / HTTP/1.1" 200 14892
[11/Oct/2023 03:28:03] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:28:03] "GET / HTTP/1.1" 200 14892
[11/Oct/2023 03:28:04] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:28:04] "GET / HTTP/1.1" 200 14892
[11/Oct/2023 03:31:56] "GET / HTTP/1.1" 200 14897
Not Found: /favicon.ico
[11/Oct/2023 03:31:56] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 03:31:57] "GET / HTTP/1.1" 200 14897
Not Found: /favicon.ico
[11/Oct/2023 03:31:57] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 03:31:58] "GET / HTTP/1.1" 200 14897
[11/Oct/2023 03:31:58] "GET / HTTP/1.1" 200 14897
[11/Oct/2023 03:34:08] "GET / HTTP/1.1" 200 14897
Not Found: /favicon.ico
[11/Oct/2023 03:34:08] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 03:35:38] "GET / HTTP/1.1" 200 23167
Not Found: /favicon.ico
[11/Oct/2023 03:35:38] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 03:36:03] "GET / HTTP/1.1" 200 23167
[11/Oct/2023 03:36:12] "GET / HTTP/1.1" 200 23167
[11/Oct/2023 03:36:13] "GET / HTTP/1.1" 200 23167
[11/Oct/2023 03:37:17] "GET / HTTP/1.1" 200 20457
[11/Oct/2023 03:37:31] "GET / HTTP/1.1" 200 20457
[11/Oct/2023 03:39:13] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 03:39:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 03:39:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 03:39:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 03:39:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 03:39:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:39:17] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 03:39:18] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:39:18] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 03:39:19] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:39:19] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 03:39:21] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 03:39:21] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:39:24] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:11] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:12] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:12] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:12] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:34] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:40:34] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:14] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:14] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:14] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:15] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:19] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:39] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 03:41:39] "GET / HTTP/1.1" 200 20445
[11/Oct/2023 03:41:41] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:41:41] "GET / HTTP/1.1" 200 20609
[11/Oct/2023 03:41:42] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:41:42] "GET / HTTP/1.1" 200 20609
[11/Oct/2023 03:41:43] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 03:41:43] "GET / HTTP/1.1" 200 20609
[11/Oct/2023 05:48:47] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 05:48:47] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 05:48:50] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 05:48:50] "GET / HTTP/1.1" 200 20284
[11/Oct/2023 05:48:52] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 05:48:53] "GET / HTTP/1.1" 200 20284
[11/Oct/2023 05:48:54] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 05:48:54] "GET / HTTP/1.1" 200 20440
[11/Oct/2023 05:48:57] "GET /add/x341115a-63d2-42ce-8fe0-5f696ecdfca6 HTTP/1.1" 302 0
[11/Oct/2023 05:48:57] "GET / HTTP/1.1" 200 20604
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 05:51:31
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 05:51:52
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 05:52:07
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 05:52:21] "GET / HTTP/1.1" 200 20604
[11/Oct/2023 05:52:22] "GET / HTTP/1.1" 200 20604
[11/Oct/2023 05:52:38] "GET / HTTP/1.1" 200 20604
[11/Oct/2023 05:52:49] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 05:52:49] "GET / HTTP/1.1" 200 20761
[11/Oct/2023 05:52:53] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 05:52:53] "GET / HTTP/1.1" 200 20761
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 05:53:33
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 05:53:33] "GET / HTTP/1.1" 200 20761
[11/Oct/2023 05:53:34] "GET / HTTP/1.1" 200 20761
[11/Oct/2023 05:55:53] "GET / HTTP/1.1" 200 20761
Not Found: /favicon.ico
[11/Oct/2023 05:55:53] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 05:55:54] "GET / HTTP/1.1" 200 20761
[11/Oct/2023 06:03:21] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 06:11:25] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:11:25] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 06:11:32] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:11:32] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 06:11:33] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:11:33] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 06:11:36] "GET /add/efe0c7a3-9835-4dfb-87e1-575b7d06701a HTTP/1.1" 302 0
[11/Oct/2023 06:11:36] "GET / HTTP/1.1" 200 20446
[11/Oct/2023 06:11:54] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 06:11:54] "GET / HTTP/1.1" 200 20603
[11/Oct/2023 06:12:40] "GET / HTTP/1.1" 200 20603
[11/Oct/2023 06:16:17] "GET / HTTP/1.1" 200 20603
[11/Oct/2023 06:18:05] "GET / HTTP/1.1" 200 20603
[11/Oct/2023 06:18:12] "GET / HTTP/1.1" 200 20603
Not Found: /favicon.ico
[11/Oct/2023 06:18:12] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 06:19:00] "GET / HTTP/1.1" 200 20603
[11/Oct/2023 06:25:55] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 06:26:06] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 06:26:06] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 06:26:08] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:26:08] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 06:26:09] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:26:09] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 06:26:09] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:26:09] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 06:26:10] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 06:26:10] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 06:26:12] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 06:26:12] "GET / HTTP/1.1" 200 20615
[11/Oct/2023 06:26:15] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 06:26:15] "GET / HTTP/1.1" 200 20615
[11/Oct/2023 06:59:07] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 06:59:08] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 07:04:16] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 07:04:16] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 07:04:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 07:04:18] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:04:26] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 07:04:26] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:04:28] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 07:04:28] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:04:28] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 07:04:28] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:04:29] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 07:04:29] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:13:57] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 07:21:12] "GET / HTTP/1.1" 200 20014
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 09:27:09
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 09:27:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 09:27:15] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 09:27:16] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 09:27:16] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 09:27:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 09:27:17] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 09:27:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 09:27:17] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 09:29:12] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 09:29:12] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 10:03:57] "GET / HTTP/1.1" 200 20168
[11/Oct/2023 10:03:57] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:03:58] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:03:58] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:04:21] "GET / HTTP/1.1" 200 20168
[11/Oct/2023 10:04:22] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:04:22] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:04:24] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:04:24] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:04:51] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:04:51] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:04:51] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:04:52] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:04:54] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:04:54] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:04:54] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:04:54] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:04:55] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:04:55] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:04:56] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:04:56] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:04:56] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:04:58] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:04:58] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:05:00] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:00] "GET / HTTP/1.1" 200 20606
[11/Oct/2023 10:05:01] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:01] "GET / HTTP/1.1" 200 20606
[11/Oct/2023 10:05:03] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:03] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:05:05] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:05] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:05:06] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:06] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:05:07] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:07] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:05:07] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:07] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:05:08] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:08] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:05:09] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:09] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:05:09] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:09] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:05:10] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:10] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:05:12] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:05:12] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:05:13] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:05:13] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:05:14] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:05:14] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:05:16] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:05:16] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:05:16] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 10:05:16] "GET / HTTP/1.1" 200 20771
[11/Oct/2023 10:05:16] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:05:16] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:05:16] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:05:16] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:05:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:05:17] "GET / HTTP/1.1" 200 20771
[11/Oct/2023 10:05:29] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 10:05:29] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:30] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:30] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:31] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:05:31] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:31] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:31] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:05:31] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:05:31] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:05:31] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:31] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:32] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:32] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:05:32] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:32] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:33] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:05:33] "GET / HTTP/1.1" 200 20607
[11/Oct/2023 10:05:36] "GET /add/42860491-9f15-43d4-adeb-0db2cc99174a HTTP/1.1" 302 0
[11/Oct/2023 10:05:36] "GET / HTTP/1.1" 200 20928
[11/Oct/2023 10:05:37] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 10:05:37] "GET / HTTP/1.1" 200 21085
[11/Oct/2023 10:05:39] "GET /add/63a3c635-4505-4588-8457-ed04fbb76511 HTTP/1.1" 302 0
[11/Oct/2023 10:05:39] "GET / HTTP/1.1" 200 21245
[11/Oct/2023 10:06:00] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:06:00] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:06:00] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 10:06:10] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:06:10] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:06:12] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:06:12] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:06:17] "GET / HTTP/1.1" 200 20606
[11/Oct/2023 10:06:18] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:06:18] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:06:21] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:06:21] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:06:23] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:06:23] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:06:25] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 10:06:25] "GET / HTTP/1.1" 200 20771
[11/Oct/2023 10:06:26] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 10:06:26] "GET / HTTP/1.1" 200 20925
[11/Oct/2023 10:06:30] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:06:30] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:06:35] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 10:06:35] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:06:37] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 10:06:38] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:06:38] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:06:38] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 10:06:40] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:06:40] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:06:45] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:06:45] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:06:45] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:06:45] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:07:38] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:07:38] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:07:40] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:07:42] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:07:43] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:07:43] "GET / HTTP/1.1" 200 21399
[11/Oct/2023 10:07:45] "GET /add/x341115a-63d2-42ce-8fe0-5f696ecdfca6 HTTP/1.1" 302 0
[11/Oct/2023 10:07:45] "GET / HTTP/1.1" 200 21563
[11/Oct/2023 10:07:47] "GET /add/x341115a-63d2-42ce-8fe0-5f696ecdfca6 HTTP/1.1" 302 0
[11/Oct/2023 10:07:47] "GET / HTTP/1.1" 200 21563
[11/Oct/2023 10:09:37] "GET / HTTP/1.1" 200 20608
[11/Oct/2023 10:13:19] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:13:19] "GET / HTTP/1.1" 200 20762
[11/Oct/2023 10:13:24] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:13:24] "GET / HTTP/1.1" 200 20762
[11/Oct/2023 10:15:31] "GET / HTTP/1.1" 200 20762
[11/Oct/2023 10:15:47] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:15:47] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:15:47] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 10:16:09] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:16:10] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:16:11] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:16:11] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:16:12] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:16:12] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:16:13] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:16:13] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:16:13] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:16:13] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:17:09] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:17:09] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:17:12] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:17:12] "GET / HTTP/1.1" 200 20294
Not Found: /favicon.ico
[11/Oct/2023 10:17:12] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:17:13] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:17:13] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:17:23] "GET / HTTP/1.1" 200 20450
[11/Oct/2023 10:20:03] "GET / HTTP/1.1" 200 20762
[11/Oct/2023 10:21:17] "GET / HTTP/1.1" 200 20294
[11/Oct/2023 10:21:46] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:24:19] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:24:19] "GET / HTTP/1.1" 200 20288
[11/Oct/2023 10:25:02] "GET / HTTP/1.1" 200 20168
[11/Oct/2023 10:25:11] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:25:11] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:25:23] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:25:23] "GET / HTTP/1.1" 200 20442
[11/Oct/2023 10:25:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:25:31] "GET / HTTP/1.1" 200 20598
[11/Oct/2023 10:28:03] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:28:03] "GET / HTTP/1.1" 200 20452
[11/Oct/2023 10:28:14] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:28:14] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:28:19] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:28:19] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:28:19] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 10:28:19] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 10:28:41] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 10:28:41] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 10:28:41] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 10:29:55] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:29:55] "GET / HTTP/1.1" 200 20286
[11/Oct/2023 10:34:32] "GET / HTTP/1.1" 200 20602
[11/Oct/2023 10:34:40] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 10:35:30] "GET / HTTP/1.1" 200 20671
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:36:12
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 10:36:18] "GET / HTTP/1.1" 200 20172
[11/Oct/2023 10:36:20] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:36:20] "GET / HTTP/1.1" 200 20504
[11/Oct/2023 10:36:22] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:36:22] "GET / HTTP/1.1" 200 20690
[11/Oct/2023 10:36:25] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:36:25] "GET / HTTP/1.1" 200 20875
[11/Oct/2023 10:37:43] "GET / HTTP/1.1" 200 20902
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:40:07
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/html/shoppingcart/index/views.py", line 102, in index
    totalSum += c['price'] * c['quantity']
TypeError: unsupported operand type(s) for +=: 'float' and 'str'
[11/Oct/2023 10:40:21] "GET / HTTP/1.1" 500 67138
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:41:20
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 10:41:22] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:41:39] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:41:39] "GET / HTTP/1.1" 200 20908
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:43:41
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 10:43:43] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:43:44] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:43:55] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:43:56] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:43:59] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:43:59] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:44:06] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:44:07] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:44:09] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 10:44:10] "GET / HTTP/1.1" 200 20172
[11/Oct/2023 10:44:27] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:44:27] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:44:27] "GET / HTTP/1.1" 200 20535
[11/Oct/2023 10:44:28] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:44:28] "GET / HTTP/1.1" 200 20535
[11/Oct/2023 10:44:30] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:44:30] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 10:44:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 10:44:31] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 10:44:33] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 10:44:34] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 10:44:41] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 10:44:41] "GET / HTTP/1.1" 200 20172
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:45:39
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:45:43
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 10:46:08] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 200 7786
[11/Oct/2023 10:46:52] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 200 7786
[11/Oct/2023 10:46:59] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 200 7786
[11/Oct/2023 10:47:00] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 200 7786
/var/www/html/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 11, 2023 - 10:47:17
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[11/Oct/2023 10:47:18] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 10:47:18] "GET / HTTP/1.1" 200 20532
[11/Oct/2023 10:47:22] "GET / HTTP/1.1" 200 20532
[11/Oct/2023 10:47:33] "GET /static/assets/dist/css/bootstrap.min.css.map HTTP/1.1" 200 589161
[11/Oct/2023 10:47:42] "GET /static/assets/dist/css/bootstrap.min.css.map HTTP/1.1" 200 589161
[11/Oct/2023 10:47:48] "GET /static/assets/dist/css/bootstrap.min.css.map HTTP/1.1" 200 589161
[11/Oct/2023 10:52:17] "GET / HTTP/1.1" 200 20532
[11/Oct/2023 10:52:54] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:52:54] "GET / HTTP/1.1" 200 20723
[11/Oct/2023 10:52:54] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:52:54] "GET / HTTP/1.1" 200 20723
[11/Oct/2023 10:52:55] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 10:52:55] "GET / HTTP/1.1" 200 20723
[11/Oct/2023 11:27:58] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:28:46] "GET / HTTP/1.1" 200 20172
[11/Oct/2023 11:29:51] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:50:25] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:50:25] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:50:25] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 11:51:14] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:51:14] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:51:14] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 11:51:14] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 11:51:36] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:51:36] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:51:36] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 11:51:43] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:51:43] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:51:43] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 11:51:43] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 11:52:42] "GET /add/efe0c7a3-9835-4dfb-87e1-575b7d06701a HTTP/1.1" 302 0
[11/Oct/2023 11:52:42] "GET / HTTP/1.1" 200 20369
[11/Oct/2023 11:52:44] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 11:52:44] "GET / HTTP/1.1" 200 20560
[11/Oct/2023 11:52:45] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:52:45] "GET / HTTP/1.1" 200 20747
[11/Oct/2023 11:52:50] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 11:52:50] "GET / HTTP/1.1" 200 20937
[11/Oct/2023 11:52:52] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 11:52:52] "GET / HTTP/1.1" 200 21121
[11/Oct/2023 11:52:58] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:53:00] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:53:00] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:53:00] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
[11/Oct/2023 11:53:01] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:53:01] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:02] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:53:02] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:03] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:53:03] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:12] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 11:53:12] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 11:53:12] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 11:53:12] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 11:53:15] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 11:53:15] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:17] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 11:53:17] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:24] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 11:53:24] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:53:25] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 11:53:25] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:54:34] "GET / HTTP/1.1" 200 20018
Not Found: /favicon.ico
[11/Oct/2023 11:54:35] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 11:54:37] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:54:38] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 11:54:40] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 11:54:40] "GET / HTTP/1.1" 200 20568
[11/Oct/2023 11:54:42] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 11:54:42] "GET / HTTP/1.1" 200 20754
[11/Oct/2023 11:54:44] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[11/Oct/2023 11:54:44] "GET / HTTP/1.1" 200 20944
[11/Oct/2023 11:54:46] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 11:54:46] "GET / HTTP/1.1" 200 21124
[11/Oct/2023 11:54:49] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 11:54:49] "GET / HTTP/1.1" 200 21124
[11/Oct/2023 11:54:50] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 11:54:51] "GET / HTTP/1.1" 200 21124
[11/Oct/2023 11:54:53] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 11:54:53] "GET / HTTP/1.1" 200 21124
[11/Oct/2023 11:54:55] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 11:54:55] "GET / HTTP/1.1" 200 21308
[11/Oct/2023 12:04:37] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 12:04:37] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 12:04:37] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 12:04:37] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 12:05:00] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 12:05:14] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 12:05:14] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 12:05:18] "GET /add/42860491-9f15-43d4-adeb-0db2cc99174a HTTP/1.1" 302 0
[11/Oct/2023 12:05:18] "GET / HTTP/1.1" 200 20562
[11/Oct/2023 12:05:20] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 12:05:20] "GET / HTTP/1.1" 200 20753
[11/Oct/2023 12:05:22] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 12:05:23] "GET / HTTP/1.1" 200 20937
Not Found: /favicon.ico
[11/Oct/2023 13:12:49] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 13:26:02] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 13:26:02] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 13:26:02] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 13:26:02] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 13:26:19] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 13:26:19] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 13:26:23] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 13:26:23] "GET / HTTP/1.1" 200 20568
[11/Oct/2023 13:26:28] "GET /add/63a3c635-4505-4588-8457-ed04fbb76511 HTTP/1.1" 302 0
[11/Oct/2023 13:26:28] "GET / HTTP/1.1" 200 20754
[11/Oct/2023 13:26:32] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[11/Oct/2023 13:26:32] "GET / HTTP/1.1" 200 20938
[11/Oct/2023 13:56:45] "GET / HTTP/1.1" 200 20018
[11/Oct/2023 13:56:50] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 13:56:50] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 14:50:40] "GET / HTTP/1.1" 200 20172
[11/Oct/2023 17:01:31] "GET / HTTP/1.1" 200 20172
Not Found: /favicon.ico
[11/Oct/2023 17:01:31] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 17:01:36] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 17:01:36] "GET / HTTP/1.1" 200 20535
[11/Oct/2023 17:01:38] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 17:01:38] "GET / HTTP/1.1" 200 20535
[11/Oct/2023 17:01:40] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 17:01:40] "GET / HTTP/1.1" 200 20722
[11/Oct/2023 17:01:43] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 17:01:43] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 17:01:46] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 17:01:46] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 17:01:49] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 17:01:49] "GET / HTTP/1.1" 200 20908
[11/Oct/2023 17:02:50] "GET / HTTP/1.1" 200 20904
[11/Oct/2023 17:22:07] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 17:22:07] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[11/Oct/2023 17:22:07] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[11/Oct/2023 17:22:07] "GET /favicon.ico HTTP/1.1" 404 2565
[11/Oct/2023 17:22:25] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 17:22:26] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 17:22:28] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[11/Oct/2023 17:22:28] "GET / HTTP/1.1" 200 20373
[11/Oct/2023 17:22:30] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 17:22:30] "GET / HTTP/1.1" 200 20564
[11/Oct/2023 17:22:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[11/Oct/2023 17:22:31] "GET / HTTP/1.1" 200 20750
[11/Oct/2023 17:22:34] "GET /add/efe0c7a3-9835-4dfb-87e1-575b7d06701a HTTP/1.1" 302 0
[11/Oct/2023 17:22:34] "GET / HTTP/1.1" 200 20929
[11/Oct/2023 18:17:46] "GET / HTTP/1.1" 200 20014
[11/Oct/2023 18:19:12] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[11/Oct/2023 18:19:12] "GET / HTTP/1.1" 200 20377
[11/Oct/2023 18:19:19] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[11/Oct/2023 18:19:19] "GET / HTTP/1.1" 200 20557
[11/Oct/2023 18:19:20] "GET /add/42860491-9f15-43d4-adeb-0db2cc99174a HTTP/1.1" 302 0
[11/Oct/2023 18:19:20] "GET / HTTP/1.1" 200 20743
[11/Oct/2023 18:36:02] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:02:06] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:02:06] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 00:02:06] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 00:02:06] "GET /favicon.ico HTTP/1.1" 404 2565
Not Found: /favicon.ico
[12/Oct/2023 00:02:06] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 00:03:16] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:03:16] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 00:03:16] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 00:03:17] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 00:03:24] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:03:25] "GET / HTTP/1.1" 200 20377
[12/Oct/2023 00:03:26] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:03:26] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:05:03] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:05:03] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 00:05:03] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 00:05:04] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 00:05:06] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 00:05:06] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 00:05:08] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:05:08] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:12:05] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:12:05] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 00:12:05] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /img/ico.png
[12/Oct/2023 00:12:06] "GET /img/ico.png HTTP/1.1" 404 2565
Not Found: /favicon.ico
[12/Oct/2023 00:12:06] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 00:12:12] "GET /add/42860491-9f15-43d4-adeb-0db2cc99174a HTTP/1.1" 302 0
[12/Oct/2023 00:12:12] "GET / HTTP/1.1" 200 20372
Not Found: /img/ico.png
[12/Oct/2023 00:12:12] "GET /img/ico.png HTTP/1.1" 404 2565
[12/Oct/2023 00:20:55] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:20:55] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 00:20:55] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 00:20:55] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 00:21:30] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:21:32] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 00:21:32] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 00:21:35] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:21:35] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:28:09] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:28:13] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:28:13] "GET / HTTP/1.1" 200 20377
[12/Oct/2023 00:28:15] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 00:28:16] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:28:17] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:28:17] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:28:23] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:28:25] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:28:56] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 00:28:56] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:28:59] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:28:59] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:29:01] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:29:02] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 00:29:51] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 00:30:01] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 00:30:01] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 00:38:25] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:25] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:38:26] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:26] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:38:26] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:26] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:38:27] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:27] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:38:27] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:27] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:38:28] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 00:38:28] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 00:46:21] "GET /add/42860491-9f15-43d4-adeb-0db2cc99174a HTTP/1.1" 302 0
[12/Oct/2023 00:46:21] "GET / HTTP/1.1" 200 20749
[12/Oct/2023 00:46:26] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:26] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:28] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:28] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:29] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:29] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:29] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:29] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:30] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:30] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:30] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:30] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:30] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:30] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:31] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:31] "GET / HTTP/1.1" 200 20935
[12/Oct/2023 00:46:32] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:32] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 00:46:32] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:32] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 00:46:33] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:33] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 00:46:33] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:33] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 00:46:34] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 00:46:34] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 00:50:50] "GET / HTTP/1.1" 200 20937
[12/Oct/2023 02:32:43] "GET / HTTP/1.1" 200 20014
Not Found: /favicon.ico
[12/Oct/2023 02:32:43] "GET /favicon.ico HTTP/1.1" 404 2565
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 12, 2023 - 02:48:18
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 12, 2023 - 02:48:26
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[12/Oct/2023 02:48:35] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 06:33:26] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 06:33:39] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 06:33:39] "GET / HTTP/1.1" 200 20373
Not Found: /favicon.ico
[12/Oct/2023 06:33:39] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 06:33:40] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 06:33:40] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 06:33:41] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 06:33:41] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 06:33:42] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 06:33:42] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 06:34:04] "GET / HTTP/1.1" 200 20168
Not Found: /favicon.ico
[12/Oct/2023 06:34:04] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 06:34:10] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 06:34:10] "GET / HTTP/1.1" 200 20531
[12/Oct/2023 06:34:11] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 06:34:11] "GET / HTTP/1.1" 200 20531
[12/Oct/2023 07:45:42] "GET / HTTP/1.1" 200 20014
Not Found: /favicon.ico
[12/Oct/2023 07:45:43] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 07:45:50] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 07:45:51] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 07:45:52] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 07:45:53] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 07:45:54] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 07:45:55] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 07:45:58] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 07:45:58] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 07:46:00] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 07:46:01] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 07:52:22] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 07:52:22] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 07:52:22] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 07:52:22] "GET /favicon.ico HTTP/1.1" 404 2565
[12/Oct/2023 08:00:42] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 08:00:43] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[12/Oct/2023 08:00:44] "GET / HTTP/1.1" 200 20373
[12/Oct/2023 08:00:45] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 08:00:46] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 08:00:47] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[12/Oct/2023 08:00:48] "GET / HTTP/1.1" 200 20564
[12/Oct/2023 08:00:49] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 08:00:49] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 08:00:50] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 08:00:51] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 08:00:53] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[12/Oct/2023 08:00:53] "GET / HTTP/1.1" 200 20750
[12/Oct/2023 08:01:49] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 08:08:47] "GET / HTTP/1.1" 200 20014
[12/Oct/2023 08:08:48] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[12/Oct/2023 08:08:48] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[12/Oct/2023 08:08:48] "GET /favicon.ico HTTP/1.1" 404 2565
[14/Oct/2023 04:00:15] "GET / HTTP/1.1" 200 20014
[14/Oct/2023 04:00:15] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[14/Oct/2023 04:00:15] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[14/Oct/2023 04:00:15] "GET /favicon.ico HTTP/1.1" 404 2565
[15/Oct/2023 23:56:44] "GET / HTTP/1.1" 200 20168
[15/Oct/2023 23:56:44] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[15/Oct/2023 23:56:44] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /apple-touch-icon-precomposed.png
[15/Oct/2023 23:56:44] "GET /apple-touch-icon-precomposed.png HTTP/1.1" 404 2628
Not Found: /apple-touch-icon.png
[15/Oct/2023 23:56:44] "GET /apple-touch-icon.png HTTP/1.1" 404 2592
[16/Oct/2023 00:01:34] "GET / HTTP/1.1" 200 20014
[16/Oct/2023 00:01:34] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[16/Oct/2023 00:01:34] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 200 232948
Not Found: /favicon.ico
[16/Oct/2023 00:01:35] "GET /favicon.ico HTTP/1.1" 404 2565
[16/Oct/2023 02:11:40] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 304 0
[16/Oct/2023 02:11:40] "GET /static/assets/js/color-modes.js HTTP/1.1" 200 2585
[16/Oct/2023 05:19:41] "GET / HTTP/1.1" 200 20014
[16/Oct/2023 05:19:41] "GET /static/assets/js/color-modes.js HTTP/1.1" 304 0
[16/Oct/2023 05:19:41] "GET /static/assets/dist/css/bootstrap.min.css HTTP/1.1" 304 0
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:17:48
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:17:53] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:17:54] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:17:55] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 07:17:55] "GET / HTTP/1.1" 200 23220
[16/Oct/2023 07:17:57] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:17:57] "GET / HTTP/1.1" 200 23435
[16/Oct/2023 07:17:58] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:17:58] "GET / HTTP/1.1" 200 23435
Internal Server Error: /make-order/
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 112, in makeorder
    insert_id = makeOrder(userid)
  File "/var/www/shoppingcart/index/application.py", line 63, in makeOrder
    return orders.insert_one(cart).inserted_id
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 671, in insert_one
    self._insert_one(
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 611, in _insert_one
    self.__database.client._retryable_write(acknowledged, _insert_command, session)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/mongo_client.py", line 1568, in _retryable_write
    return self._retry_with_session(retryable, func, s, None)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/mongo_client.py", line 1413, in _retry_with_session
    return self._retry_internal(retryable, func, session, bulk)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/_csot.py", line 108, in csot_wrapper
    return func(self, *args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/mongo_client.py", line 1460, in _retry_internal
    return func(session, conn, retryable)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 599, in _insert_command
    result = conn.command(
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/helpers.py", line 315, in inner
    return func(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/pool.py", line 932, in command
    return command(
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/network.py", line 191, in command
    helpers._check_command_response(
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/helpers.py", line 223, in _check_command_response
    raise OperationFailure(errmsg, code, response, max_wire_version)
pymongo.errors.OperationFailure: not authorized on ShoppingCart to execute command { insert: "orders", ordered: true, lsid: { id: UUID("587e90f3-20bb-4925-9719-db5b773c9a96") }, $db: "ShoppingCart" }, full error: {'ok': 0.0, 'errmsg': 'not authorized on ShoppingCart to execute command { insert: "orders", ordered: true, lsid: { id: UUID("587e90f3-20bb-4925-9719-db5b773c9a96") }, $db: "ShoppingCart" }', 'code': 13, 'codeName': 'Unauthorized'}
[16/Oct/2023 07:18:00] "GET /make-order/ HTTP/1.1" 500 138462
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:18:23
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:18:26] "GET / HTTP/1.1" 200 23435
[16/Oct/2023 07:18:27] "GET /make-order/ HTTP/1.1" 200 7783
[16/Oct/2023 07:18:32] "GET /make-order/ HTTP/1.1" 200 7783
[16/Oct/2023 07:18:33] "GET /make-order/ HTTP/1.1" 200 7783
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:19:54
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:23:03] "GET / HTTP/1.1" 200 23435
[16/Oct/2023 07:23:05] "GET /make-order/ HTTP/1.1" 200 7783
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 3, in <module>
    from . import views
  File "/var/www/shoppingcart/index/views.py", line 9, in <module>
    from application import *
  File "/var/www/shoppingcart/index/application.py", line 61
    order['date'] = 
                    ^
SyntaxError: invalid syntax
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 3, in <module>
    from . import views
  File "/var/www/shoppingcart/index/views.py", line 9, in <module>
    from application import *
  File "/var/www/shoppingcart/index/application.py", line 62
    order['date'] =
                   ^
SyntaxError: invalid syntax
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:25:11
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:25:22
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:28:02
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:28:24] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:28:25] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:28:26] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:28:26] "GET / HTTP/1.1" 200 23224
[16/Oct/2023 07:28:28] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:28:28] "GET / HTTP/1.1" 200 23224
[16/Oct/2023 07:28:29] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[16/Oct/2023 07:28:29] "GET / HTTP/1.1" 200 23428
[16/Oct/2023 07:28:31] "GET /make-order/ HTTP/1.1" 200 7404
[16/Oct/2023 07:29:14] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:29:15] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:29:17] "GET /add/63a3c635-4505-4588-8457-ed04fbb76511 HTTP/1.1" 302 0
[16/Oct/2023 07:29:17] "GET / HTTP/1.1" 200 23218
[16/Oct/2023 07:29:18] "GET /add/63a3c635-4505-4588-8457-ed04fbb76511 HTTP/1.1" 302 0
[16/Oct/2023 07:29:18] "GET / HTTP/1.1" 200 23219
[16/Oct/2023 07:29:19] "GET /add/63a3c635-4505-4588-8457-ed04fbb76511 HTTP/1.1" 302 0
[16/Oct/2023 07:29:19] "GET / HTTP/1.1" 200 23219
[16/Oct/2023 07:29:20] "GET /make-order/ HTTP/1.1" 200 7404
[16/Oct/2023 07:29:24] "GET / HTTP/1.1" 200 21990
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:29:53
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:30:16] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:30:17] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:30:17] "GET / HTTP/1.1" 200 23224
[16/Oct/2023 07:30:19] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[16/Oct/2023 07:30:20] "GET / HTTP/1.1" 200 23432
[16/Oct/2023 07:30:21] "GET /add/f5384efc-eadb-4d7b-a131-36516269c218 HTTP/1.1" 302 0
[16/Oct/2023 07:30:21] "GET / HTTP/1.1" 200 23646
[16/Oct/2023 07:30:23] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 07:30:23] "GET / HTTP/1.1" 200 21990
[16/Oct/2023 07:32:07] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:32:07] "GET / HTTP/1.1" 200 23224
[16/Oct/2023 07:32:08] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 07:32:08] "GET / HTTP/1.1" 200 23224
[16/Oct/2023 07:34:08] "GET / HTTP/1.1" 200 23350
[16/Oct/2023 07:35:47] "GET / HTTP/1.1" 200 23542
[16/Oct/2023 07:38:37] "GET / HTTP/1.1" 200 23725
[16/Oct/2023 07:38:47] "GET / HTTP/1.1" 200 23727
[16/Oct/2023 07:39:09] "GET / HTTP/1.1" 200 23752
[16/Oct/2023 07:40:29] "GET / HTTP/1.1" 200 23703
[16/Oct/2023 07:40:47] "GET / HTTP/1.1" 200 23718
Not Found: /favicon.ico
[16/Oct/2023 07:40:47] "GET /favicon.ico HTTP/1.1" 404 2741
[16/Oct/2023 07:41:00] "GET / HTTP/1.1" 200 23748
[16/Oct/2023 07:41:06] "GET / HTTP/1.1" 200 23742
[16/Oct/2023 07:41:13] "GET / HTTP/1.1" 200 23735
Not Found: /favicon.ico
[16/Oct/2023 07:41:13] "GET /favicon.ico HTTP/1.1" 404 2741
[16/Oct/2023 07:41:22] "GET / HTTP/1.1" 200 23727
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 3, in <module>
    from . import views
  File "/var/www/shoppingcart/index/views.py", line 90
    request.session['username'] = $_POST['username']
                                  ^
SyntaxError: invalid syntax
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:45:48
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:46:45
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:47:03
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:48:20
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 96, in index
    'username': request.session['username'],
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'username'
[16/Oct/2023 07:48:57] "GET / HTTP/1.1" 500 66316
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 96, in index
    'username': request.session['username'],
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'username'
[16/Oct/2023 07:48:58] "GET / HTTP/1.1" 500 66316
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:50:08
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:50:10] "GET / HTTP/1.1" 200 22348
Not Found: /index/
[16/Oct/2023 07:50:15] "POST /index/ HTTP/1.1" 404 2727
[16/Oct/2023 07:51:16] "GET / HTTP/1.1" 200 22342
Forbidden (CSRF token missing.): /
[16/Oct/2023 07:51:17] "POST / HTTP/1.1" 403 2506
[16/Oct/2023 07:51:19] "GET / HTTP/1.1" 200 22342
[16/Oct/2023 07:51:19] "GET / HTTP/1.1" 200 22342
Forbidden (CSRF token missing.): /
[16/Oct/2023 07:51:20] "POST / HTTP/1.1" 403 2506
[16/Oct/2023 07:51:23] "GET / HTTP/1.1" 200 22342
[16/Oct/2023 07:51:45] "GET / HTTP/1.1" 200 22476
[16/Oct/2023 07:51:50] "POST / HTTP/1.1" 200 22476
[16/Oct/2023 07:51:56] "POST / HTTP/1.1" 200 22476
[16/Oct/2023 07:51:59] "POST / HTTP/1.1" 200 22476
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:52:22
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:52:25] "GET / HTTP/1.1" 200 22476
[16/Oct/2023 07:52:26] "GET / HTTP/1.1" 200 22476
[16/Oct/2023 07:52:29] "POST / HTTP/1.1" 200 22476
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:53:45
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:54:23] "GET / HTTP/1.1" 200 22473
[16/Oct/2023 07:54:24] "GET / HTTP/1.1" 200 22473
[16/Oct/2023 07:54:27] "POST / HTTP/1.1" 200 22473
[16/Oct/2023 07:54:31] "POST / HTTP/1.1" 200 22473
[16/Oct/2023 07:54:43] "POST / HTTP/1.1" 200 22474
[16/Oct/2023 07:54:44] "GET / HTTP/1.1" 200 22474
[16/Oct/2023 07:54:45] "GET / HTTP/1.1" 200 22474
[16/Oct/2023 07:54:48] "POST / HTTP/1.1" 200 22474
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:55:21
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 9
    path('make-order/', views.makeorder, name="makeorder")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 879, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1017, in get_code
  File "<frozen importlib._bootstrap_external>", line 947, in source_to_code
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 9
    path('make-order/', views.makeorder, name="makeorder")
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:55:55
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 07:57:30
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 07:57:36] "GET / HTTP/1.1" 200 22492
[16/Oct/2023 07:57:37] "GET / HTTP/1.1" 200 22492
Not Found: /savename/
[16/Oct/2023 07:57:42] "POST /savename/ HTTP/1.1" 404 2910
Not Found: /savename/
[16/Oct/2023 07:57:53] "POST /savename/ HTTP/1.1" 404 2910
[16/Oct/2023 07:57:57] "GET / HTTP/1.1" 200 22483
[16/Oct/2023 07:57:59] "GET / HTTP/1.1" 200 22483
[16/Oct/2023 07:58:02] "POST / HTTP/1.1" 200 22478
[16/Oct/2023 07:58:07] "GET / HTTP/1.1" 200 22483
[16/Oct/2023 07:58:09] "GET / HTTP/1.1" 200 22483
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:00:41
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 98, in index
    'username': username,
NameError: name 'username' is not defined
[16/Oct/2023 08:00:46] "GET / HTTP/1.1" 500 62340
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:00:59
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:00:59] "GET / HTTP/1.1" 200 22483
[16/Oct/2023 08:01:00] "GET / HTTP/1.1" 200 22483
[16/Oct/2023 08:01:02] "POST / HTTP/1.1" 200 22477
[16/Oct/2023 08:01:04] "GET / HTTP/1.1" 200 22477
[16/Oct/2023 08:01:07] "GET / HTTP/1.1" 200 22477
[16/Oct/2023 08:01:11] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:01:11] "GET / HTTP/1.1" 200 23829
[16/Oct/2023 08:01:13] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:01:13] "GET / HTTP/1.1" 200 24044
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 95, in index
    return render(request, 'base.html',
  File "/home/gulden/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' !== 'Undefined' ? username : ''' from 'username !== 'Undefined' ? username : '''
[16/Oct/2023 08:02:10] "GET / HTTP/1.1" 500 162363
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 95, in index
    return render(request, 'base.html',
  File "/home/gulden/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' username !== 'Undefined'' from 'if username !== 'Undefined''
[16/Oct/2023 08:05:36] "GET / HTTP/1.1" 500 164086
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 95, in index
    return render(request, 'base.html',
  File "/home/gulden/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' username !== 'Undefined'' from 'if username !== 'Undefined''
[16/Oct/2023 08:05:37] "GET / HTTP/1.1" 500 163940
Not Found: /favicon.ico
[16/Oct/2023 08:05:37] "GET /favicon.ico HTTP/1.1" 404 2915
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 95, in index
    return render(request, 'base.html',
  File "/home/gulden/.local/lib/python3.10/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/backends/django.py", line 33, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 175, in get_template
    template, origin = self.find_template(template_name)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/loaders/base.py", line 28, in get_template
    return Template(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 200, in compile_nodelist
    return parser.parse()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 484, in parse
    raise self.error(token, e)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 482, in parse
    filter_expression = self.compile_filter(token.contents)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 600, in compile_filter
    return FilterExpression(token, self)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/template/base.py", line 703, in __init__
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: ' username == 'Undefined'' from 'if username == 'Undefined''
[16/Oct/2023 08:05:48] "GET / HTTP/1.1" 500 163923
[16/Oct/2023 08:06:14] "GET / HTTP/1.1" 200 23759
[16/Oct/2023 08:06:19] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:06:19] "GET / HTTP/1.1" 200 23759
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:06:55
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:07:19
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:07:40
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:07:41] "GET / HTTP/1.1" 200 23759
Not Found: /favicon.ico
[16/Oct/2023 08:07:41] "GET /favicon.ico HTTP/1.1" 404 2915
[16/Oct/2023 08:07:41] "GET / HTTP/1.1" 200 23759
[16/Oct/2023 08:07:44] "GET /make-order HTTP/1.1" 301 0
[16/Oct/2023 08:07:44] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:07:44] "GET / HTTP/1.1" 200 22192
[16/Oct/2023 08:08:14] "GET / HTTP/1.1" 200 22192
[16/Oct/2023 08:08:15] "GET / HTTP/1.1" 200 22192
[16/Oct/2023 08:09:42] "GET / HTTP/1.1" 200 22474
Not Found: /favicon.ico
[16/Oct/2023 08:09:42] "GET /favicon.ico HTTP/1.1" 404 2915
[16/Oct/2023 08:09:46] "POST / HTTP/1.1" 200 22474
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:11:20
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:12:05
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:12:53
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:13:22] "GET / HTTP/1.1" 200 22487
Internal Server Error: /save-name/edit
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 204, in _get_response
    self.check_response(response, callback)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 332, in check_response
    raise ValueError(
ValueError: The view index.views.savename didn't return an HttpResponse object. It returned None instead.
[16/Oct/2023 08:13:23] "POST /save-name/edit HTTP/1.1" 500 62105
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:13:44
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:13:47] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:13:47] "GET / HTTP/1.1" 200 22484
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:14:14
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:14:15] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:14:15] "GET / HTTP/1.1" 200 22519
[16/Oct/2023 08:14:19] "POST / HTTP/1.1" 200 22489
[16/Oct/2023 08:14:21] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:14:21] "GET / HTTP/1.1" 200 22519
[16/Oct/2023 08:14:24] "POST / HTTP/1.1" 200 22489
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:15:13
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:15:38
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:15:51
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:16:20
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:16:23] "GET / HTTP/1.1" 200 22489
[16/Oct/2023 08:16:25] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:16:25] "GET / HTTP/1.1" 200 22528
Not Found: /save-name/
[16/Oct/2023 08:16:29] "POST /save-name/ HTTP/1.1" 404 2927
Not Found: /save-name/
[16/Oct/2023 08:17:06] "POST /save-name/ HTTP/1.1" 404 2927
[16/Oct/2023 08:17:36] "GET / HTTP/1.1" 200 22529
[16/Oct/2023 08:17:36] "GET / HTTP/1.1" 200 22529
[16/Oct/2023 08:17:41] "GET / HTTP/1.1" 200 22529
Not Found: /save-name/
[16/Oct/2023 08:17:44] "POST /save-name/ HTTP/1.1" 404 2927
Not Found: /save-name/
[16/Oct/2023 08:18:55] "POST /save-name/ HTTP/1.1" 404 2927
[16/Oct/2023 08:18:58] "GET / HTTP/1.1" 200 22533
[16/Oct/2023 08:19:00] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:19:00] "GET / HTTP/1.1" 200 22489
[16/Oct/2023 08:19:02] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:19:02] "GET / HTTP/1.1" 200 22533
[16/Oct/2023 08:19:05] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:19:05] "GET / HTTP/1.1" 200 22491
[16/Oct/2023 08:19:08] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:19:08] "GET / HTTP/1.1" 200 22533
[16/Oct/2023 08:19:10] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:19:10] "GET / HTTP/1.1" 200 22493
[16/Oct/2023 08:19:11] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:19:11] "GET / HTTP/1.1" 200 22533
[16/Oct/2023 08:19:13] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:19:14] "GET / HTTP/1.1" 200 22490
[16/Oct/2023 08:19:15] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:19:15] "GET / HTTP/1.1" 200 23846
[16/Oct/2023 08:19:17] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:19:17] "GET / HTTP/1.1" 200 23846
[16/Oct/2023 08:19:18] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:19:18] "GET / HTTP/1.1" 200 22490
[16/Oct/2023 08:19:43] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:19:43] "GET / HTTP/1.1" 200 23842
[16/Oct/2023 08:19:44] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:19:44] "GET / HTTP/1.1" 200 23842
[16/Oct/2023 08:25:17] "GET / HTTP/1.1" 200 24142
[16/Oct/2023 08:25:27] "GET / HTTP/1.1" 200 24142
[16/Oct/2023 08:25:55] "GET / HTTP/1.1" 200 24144
[16/Oct/2023 08:25:55] "GET / HTTP/1.1" 200 24144
[16/Oct/2023 08:26:21] "GET / HTTP/1.1" 200 24149
[16/Oct/2023 08:26:57] "GET / HTTP/1.1" 200 24149
[16/Oct/2023 08:26:59] "GET / HTTP/1.1" 200 23849
[16/Oct/2023 08:27:11] "GET / HTTP/1.1" 200 23849
[16/Oct/2023 08:27:12] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:27:30] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:27:43] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:27:48] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:27:56] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:27:56] "GET / HTTP/1.1" 200 23845
[16/Oct/2023 08:28:16] "GET / HTTP/1.1" 200 23820
[16/Oct/2023 08:28:17] "GET / HTTP/1.1" 200 23820
[16/Oct/2023 08:28:46] "GET / HTTP/1.1" 200 24027
[16/Oct/2023 08:29:25] "GET / HTTP/1.1" 200 24138
[16/Oct/2023 08:30:00] "GET / HTTP/1.1" 200 22649
[16/Oct/2023 08:30:01] "GET / HTTP/1.1" 200 22649
[16/Oct/2023 08:30:01] "GET / HTTP/1.1" 200 22649
[16/Oct/2023 08:30:10] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:30:10] "GET / HTTP/1.1" 200 24138
[16/Oct/2023 08:30:13] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:30:13] "GET / HTTP/1.1" 200 24138
[16/Oct/2023 08:30:15] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:30:15] "GET / HTTP/1.1" 200 24381
[16/Oct/2023 08:30:17] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:30:17] "GET / HTTP/1.1" 200 24381
[16/Oct/2023 08:30:26] "GET / HTTP/1.1" 200 24381
[16/Oct/2023 08:30:27] "GET / HTTP/1.1" 200 24381
[16/Oct/2023 08:30:28] "GET / HTTP/1.1" 200 24381
[16/Oct/2023 08:30:56] "GET / HTTP/1.1" 200 24490
[16/Oct/2023 08:31:04] "GET / HTTP/1.1" 200 24490
[16/Oct/2023 08:31:17] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:31:17] "GET / HTTP/1.1" 200 24533
[16/Oct/2023 08:31:22] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:31:22] "GET / HTTP/1.1" 200 24491
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

Exception in thread django-main-thread:
Traceback (most recent call last):
  File "/usr/lib/python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/usr/lib/python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/autoreload.py", line 64, in wrapper
    fn(*args, **kwargs)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/commands/runserver.py", line 133, in inner_run
    self.check(display_num_errors=True)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/management/base.py", line 485, in check
    all_issues = checks.run_checks(
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 494, in check
    for pattern in self.url_patterns:
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 715, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/utils/functional.py", line 57, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/resolvers.py", line 708, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/shoppingcart/urls.py", line 21, in <module>
    path("", include("index.urls")),
  File "/home/gulden/.local/lib/python3.10/site-packages/django/urls/conf.py", line 38, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/var/www/shoppingcart/index/urls.py", line 3, in <module>
    from . import views
  File "/var/www/shoppingcart/index/views.py", line 9, in <module>
    from application import *
  File "/var/www/shoppingcart/index/application.py", line 75
    user_orders =
                 ^
SyntaxError: invalid syntax
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:33:33
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:34:09
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(uid)
NameError: name 'uid' is not defined
[16/Oct/2023 08:34:57] "GET / HTTP/1.1" 500 62546
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:35:15
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 75, in getOrdersByUserId
    user_orders = mdb.find({"userid": uid})
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 3503, in __call__
    raise TypeError(
TypeError: 'Collection' object is not callable. If you meant to call the 'find' method on a 'Database' object it is failing because no such method exists.
[16/Oct/2023 08:35:17] "GET / HTTP/1.1" 500 70489
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 75, in getOrdersByUserId
    user_orders = mdb.find({"userid": uid})
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 3503, in __call__
    raise TypeError(
TypeError: 'Collection' object is not callable. If you meant to call the 'find' method on a 'Database' object it is failing because no such method exists.
[16/Oct/2023 08:35:18] "GET / HTTP/1.1" 500 70489
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:36:05
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 75, in getOrdersByUserId
    user_orders = mdb.find()
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 3503, in __call__
    raise TypeError(
TypeError: 'Collection' object is not callable. If you meant to call the 'find' method on a 'Database' object it is failing because no such method exists.
[16/Oct/2023 08:36:06] "GET / HTTP/1.1" 500 70372
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 75, in getOrdersByUserId
    user_orders = mdb.find()
  File "/home/gulden/.local/lib/python3.10/site-packages/pymongo/collection.py", line 3503, in __call__
    raise TypeError(
TypeError: 'Collection' object is not callable. If you meant to call the 'find' method on a 'Database' object it is failing because no such method exists.
[16/Oct/2023 08:36:07] "GET / HTTP/1.1" 500 70372
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:36:38
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:36:38] "GET / HTTP/1.1" 200 24558
[16/Oct/2023 08:36:41] "GET / HTTP/1.1" 200 24558
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:36:53
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:36:53] "GET / HTTP/1.1" 200 24558
[16/Oct/2023 08:36:54] "GET / HTTP/1.1" 200 24558
[16/Oct/2023 08:36:55] "GET / HTTP/1.1" 200 24558
[16/Oct/2023 08:39:32] "GET / HTTP/1.1" 200 24504
[16/Oct/2023 08:39:33] "GET / HTTP/1.1" 200 24504
[16/Oct/2023 08:43:11] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:43:13] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:43:14] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:43:55] "GET / HTTP/1.1" 200 22663
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:44:02
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 08:44:02] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:44:03] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:44:04] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:44:04] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:44:04] "GET / HTTP/1.1" 200 22663
[16/Oct/2023 08:44:23] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:44:24] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:44:25] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:44:25] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:44:25] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:46:27] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:46:28] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:46:29] "GET / HTTP/1.1" 200 22730
[16/Oct/2023 08:46:30] "GET / HTTP/1.1" 200 22730
/var/www/shoppingcart/index/application.py changed, reloading.
[16/Oct/2023 08:46:44] "GET / HTTP/1.1" 200 22730
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:46:44
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

<pymongo.cursor.Cursor object at 0x7f27817091b0>
[16/Oct/2023 08:46:44] "GET / HTTP/1.1" 200 22730
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 08:48:12
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 78, in getOrdersByUserId
    r.append({'date': o['date'], 'total': o['total']})
KeyError: 'date'
[16/Oct/2023 08:48:14] "GET / HTTP/1.1" 500 66728
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    orders = getOrdersByUserId(userid)
  File "/var/www/shoppingcart/index/application.py", line 78, in getOrdersByUserId
    r.append({'date': o['date'], 'total': o['total']})
KeyError: 'date'
[16/Oct/2023 08:48:50] "GET / HTTP/1.1" 500 66728
[16/Oct/2023 08:49:23] "GET / HTTP/1.1" 200 22678
[16/Oct/2023 08:49:28] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:49:28] "GET / HTTP/1.1" 200 24280
[16/Oct/2023 08:49:29] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:49:29] "GET / HTTP/1.1" 200 24280
[16/Oct/2023 08:49:31] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 08:49:31] "GET / HTTP/1.1" 200 24519
[16/Oct/2023 08:49:35] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:49:35] "GET / HTTP/1.1" 200 23049
[16/Oct/2023 08:49:47] "GET / HTTP/1.1" 200 22956
[16/Oct/2023 08:49:51] "GET /add/efe0c7a3-9835-4dfb-87e1-575b7d06701a HTTP/1.1" 302 0
[16/Oct/2023 08:49:51] "GET / HTTP/1.1" 200 24546
[16/Oct/2023 08:49:53] "GET /add/efe0c7a3-9835-4dfb-87e1-575b7d06701a HTTP/1.1" 302 0
[16/Oct/2023 08:49:53] "GET / HTTP/1.1" 200 24546
[16/Oct/2023 08:49:55] "GET /add/97a19842-db31-4537-9241-5053d7c96239 HTTP/1.1" 302 0
[16/Oct/2023 08:49:55] "GET / HTTP/1.1" 200 24782
[16/Oct/2023 08:49:56] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:49:56] "GET / HTTP/1.1" 200 23095
[16/Oct/2023 08:50:12] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:50:12] "GET / HTTP/1.1" 200 24693
[16/Oct/2023 08:50:14] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:50:14] "GET / HTTP/1.1" 200 24693
[16/Oct/2023 08:50:32] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:50:32] "GET / HTTP/1.1" 200 24735
[16/Oct/2023 08:50:36] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:50:36] "GET / HTTP/1.1" 200 24691
[16/Oct/2023 08:50:39] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:50:39] "GET / HTTP/1.1" 200 23233
[16/Oct/2023 08:50:46] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 08:50:46] "GET / HTTP/1.1" 200 24831
[16/Oct/2023 08:50:48] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 08:50:48] "GET / HTTP/1.1" 200 24831
[16/Oct/2023 08:50:52] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:50:52] "GET / HTTP/1.1" 200 23374
[16/Oct/2023 08:51:54] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[16/Oct/2023 08:51:54] "GET / HTTP/1.1" 200 24964
Not Found: /favicon.ico
[16/Oct/2023 08:51:54] "GET /favicon.ico HTTP/1.1" 404 2929
[16/Oct/2023 08:51:57] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:51:57] "GET / HTTP/1.1" 200 23512
[16/Oct/2023 08:53:03] "GET / HTTP/1.1" 200 23632
[16/Oct/2023 08:53:08] "GET / HTTP/1.1" 200 22712
[16/Oct/2023 08:53:28] "GET / HTTP/1.1" 200 22729
[16/Oct/2023 08:54:04] "GET / HTTP/1.1" 200 22743
[16/Oct/2023 08:54:13] "GET / HTTP/1.1" 200 22757
[16/Oct/2023 08:54:20] "GET / HTTP/1.1" 200 22757
[16/Oct/2023 08:54:21] "GET / HTTP/1.1" 200 22762
[16/Oct/2023 08:54:27] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:42] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:43] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:43] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:43] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:44] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:44] "GET / HTTP/1.1" 200 22767
[16/Oct/2023 08:54:54] "GET / HTTP/1.1" 200 22773
[16/Oct/2023 08:54:55] "GET / HTTP/1.1" 200 22773
[16/Oct/2023 08:54:55] "GET / HTTP/1.1" 200 22773
[16/Oct/2023 08:54:55] "GET / HTTP/1.1" 200 22773
[16/Oct/2023 08:55:22] "GET / HTTP/1.1" 200 22786
[16/Oct/2023 08:55:37] "POST /save-name/edit HTTP/1.1" 302 0
[16/Oct/2023 08:55:37] "GET / HTTP/1.1" 200 22811
[16/Oct/2023 08:55:41] "POST /save-name/save HTTP/1.1" 302 0
[16/Oct/2023 08:55:41] "GET / HTTP/1.1" 200 22787
[16/Oct/2023 08:55:43] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:55:43] "GET / HTTP/1.1" 200 24389
[16/Oct/2023 08:55:46] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:55:46] "GET / HTTP/1.1" 200 24389
[16/Oct/2023 08:55:48] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 08:55:48] "GET / HTTP/1.1" 200 24628
[16/Oct/2023 08:55:50] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:55:50] "GET / HTTP/1.1" 200 23052
[16/Oct/2023 08:56:48] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 08:56:48] "GET / HTTP/1.1" 200 24650
[16/Oct/2023 08:56:49] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:56:49] "GET / HTTP/1.1" 200 24888
[16/Oct/2023 08:56:51] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:56:52] "GET / HTTP/1.1" 200 23213
[16/Oct/2023 08:57:48] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 08:57:48] "GET / HTTP/1.1" 200 24815
[16/Oct/2023 08:57:49] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:57:49] "GET / HTTP/1.1" 200 25054
[16/Oct/2023 08:57:51] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:57:51] "GET / HTTP/1.1" 200 23373
[16/Oct/2023 08:58:15] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:58:16] "GET / HTTP/1.1" 200 24971
[16/Oct/2023 08:58:17] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 08:58:17] "GET / HTTP/1.1" 200 24971
[16/Oct/2023 08:58:20] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 08:58:20] "GET / HTTP/1.1" 200 23533
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:02:20
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:03:42
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:03:59
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 09:04:01] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 09:04:01] "GET / HTTP/1.1" 200 25135
[16/Oct/2023 09:04:03] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 09:04:03] "GET / HTTP/1.1" 200 23692
/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:05:06
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 09:05:08] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 09:05:08] "GET / HTTP/1.1" 200 25294
[16/Oct/2023 09:05:09] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 09:05:09] "GET / HTTP/1.1" 200 23851
[16/Oct/2023 09:05:32] "GET / HTTP/1.1" 200 22787
[16/Oct/2023 09:05:34] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 09:05:34] "GET / HTTP/1.1" 200 24389
[16/Oct/2023 09:05:35] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 09:05:35] "GET / HTTP/1.1" 200 24389
[16/Oct/2023 09:05:37] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 09:05:37] "GET / HTTP/1.1" 200 24628
[16/Oct/2023 09:05:40] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 09:05:40] "GET / HTTP/1.1" 200 23052
[16/Oct/2023 09:05:46] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 09:05:46] "GET / HTTP/1.1" 200 24650
[16/Oct/2023 09:05:48] "GET /add/e182115a-63d2-42ce-8fe0-5f696ecdfba6 HTTP/1.1" 302 0
[16/Oct/2023 09:05:48] "GET / HTTP/1.1" 200 24650
[16/Oct/2023 09:05:49] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 09:05:49] "GET / HTTP/1.1" 200 23212
[16/Oct/2023 09:07:11] "GET / HTTP/1.1" 200 23215
[16/Oct/2023 09:07:42] "GET / HTTP/1.1" 200 23219
Not Found: /favicon.ico
[16/Oct/2023 09:07:42] "GET /favicon.ico HTTP/1.1" 404 2929
[16/Oct/2023 09:07:50] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[16/Oct/2023 09:07:50] "GET / HTTP/1.1" 200 24809
[16/Oct/2023 09:07:52] "GET /add/6d6ca89d-fbc2-4fc2-93d0-6ee46ae97345 HTTP/1.1" 302 0
[16/Oct/2023 09:07:52] "GET / HTTP/1.1" 200 24810
[16/Oct/2023 09:07:55] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 09:07:55] "GET / HTTP/1.1" 200 23378
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:11:33
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:11:45
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:12:03
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Not Found: /
[16/Oct/2023 09:14:46] "GET / HTTP/1.1" 404 2893
[16/Oct/2023 09:14:46] "GET /favicon.ico HTTP/1.1" 200 23661
Not Found: /
[16/Oct/2023 09:14:59] "GET / HTTP/1.1" 404 2893
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified some issues:

WARNINGS:
?: (urls.W002) Your URL pattern '/<message>' [name='index'] has a route beginning with a '/'. Remove this slash as it is unnecessary. If this pattern is targeted in an include(), ensure the include() pattern has a trailing '/'.

System check identified 1 issue (0 silenced).
October 16, 2023 - 09:15:12
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Not Found: /
[16/Oct/2023 09:15:13] "GET / HTTP/1.1" 404 2894
Not Found: /
[16/Oct/2023 09:15:14] "GET / HTTP/1.1" 404 2894
/var/www/shoppingcart/index/urls.py changed, reloading.
Not Found: /
[16/Oct/2023 09:15:18] "GET / HTTP/1.1" 404 2894
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:15:18
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 09:15:19] "GET / HTTP/1.1" 200 23319
[16/Oct/2023 09:15:20] "GET / HTTP/1.1" 200 23319
[16/Oct/2023 09:15:20] "GET / HTTP/1.1" 200 23319
[16/Oct/2023 09:15:20] "GET / HTTP/1.1" 200 23319
Not Found: /sss
[16/Oct/2023 09:15:27] "GET /sss HTTP/1.1" 404 2905
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified some issues:

WARNINGS:
?: (2_0.W001) Your URL pattern '^$' [name='index'] has a route that contains '(?P<', begins with a '^', or ends with a '$'. This was likely an oversight when migrating to django.urls.path().

System check identified 1 issue (0 silenced).
October 16, 2023 - 09:17:08
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified some issues:

WARNINGS:
?: (2_0.W001) Your URL pattern '^$' [name='index'] has a route that contains '(?P<', begins with a '^', or ends with a '$'. This was likely an oversight when migrating to django.urls.path().

System check identified 1 issue (0 silenced).
October 16, 2023 - 09:17:13
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Not Found: /
[16/Oct/2023 09:17:15] "GET / HTTP/1.1" 404 2880
Not Found: /
[16/Oct/2023 09:17:16] "GET / HTTP/1.1" 404 2880
Not Found: /
[16/Oct/2023 09:17:16] "GET / HTTP/1.1" 404 2880
/var/www/shoppingcart/index/urls.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:17:26
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 09:17:33] "GET / HTTP/1.1" 200 23319
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:17:58
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 101, in index
    'message': message})
NameError: name 'message' is not defined
[16/Oct/2023 09:18:02] "GET / HTTP/1.1" 500 62930
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 101, in index
    'message': message})
NameError: name 'message' is not defined
[16/Oct/2023 09:18:02] "GET / HTTP/1.1" 500 62930
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 101, in index
    'message': message})
NameError: name 'message' is not defined
[16/Oct/2023 09:18:03] "GET / HTTP/1.1" 500 62930
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 101, in index
    'message': message})
NameError: name 'message' is not defined
[16/Oct/2023 09:18:03] "GET / HTTP/1.1" 500 62930
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:18:24
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:18:37
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 09:18:37] "GET / HTTP/1.1" 200 23661
[16/Oct/2023 09:18:38] "GET / HTTP/1.1" 200 23661
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:58:07
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:58:28
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 09:59:47
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/application.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:03:02
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 102, in index
    'message': showMessage()
  File "/var/www/shoppingcart/index/application.py", line 89, in showMessage
    message = requests.session['message']
TypeError: 'function' object is not subscriptable
[16/Oct/2023 10:04:11] "GET / HTTP/1.1" 500 65766
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:07:09
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:07:27
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 102, in index
    'message': request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:07:27] "GET / HTTP/1.1" 500 66664
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 102, in index
    'message': request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:07:28] "GET / HTTP/1.1" 500 66664
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:08:16
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 102, in index
    'message': request.session['message'],
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:08:17] "GET / HTTP/1.1" 500 66666
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:09:10
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 10:09:11] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:09:15] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:09:16] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:09:16] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:09:17] "GET / HTTP/1.1" 200 23623
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:09:53
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = showMessage()
  File "/var/www/shoppingcart/index/application.py", line 89, in showMessage
    message = requests.session['message']
TypeError: 'function' object is not subscriptable
[16/Oct/2023 10:09:53] "GET / HTTP/1.1" 500 63407
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = showMessage()
  File "/var/www/shoppingcart/index/application.py", line 89, in showMessage
    message = requests.session['message']
TypeError: 'function' object is not subscriptable
[16/Oct/2023 10:09:54] "GET / HTTP/1.1" 500 63407
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = request.session['message']
  File "/var/www/shoppingcart/index/application.py", line 89, in showMessage
    message = requests.session['message']
TypeError: 'function' object is not subscriptable
[16/Oct/2023 10:10:09] "GET / HTTP/1.1" 500 63453
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:10:10
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:10] "GET / HTTP/1.1" 500 64305
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:10] "GET / HTTP/1.1" 500 64305
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:11] "GET / HTTP/1.1" 500 64305
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    message = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:11] "GET / HTTP/1.1" 500 64305
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:10:29
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:29] "GET / HTTP/1.1" 500 64297
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:30] "GET / HTTP/1.1" 500 64297
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:30] "GET / HTTP/1.1" 500 64297
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['message']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'message'
[16/Oct/2023 10:10:30] "GET / HTTP/1.1" 500 64297
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:11:07
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:07] "GET / HTTP/1.1" 500 64269
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:08] "GET / HTTP/1.1" 500 64269
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:11:26
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:27] "GET / HTTP/1.1" 500 64265
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:27] "GET / HTTP/1.1" 500 64265
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:28] "GET / HTTP/1.1" 500 64265
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:28] "GET / HTTP/1.1" 500 64265
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:28] "GET / HTTP/1.1" 500 64265
/var/www/shoppingcart/index/views.py changed, reloading.
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = 'xxx'
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:11:56] "GET / HTTP/1.1" 500 64231
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:11:56
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 10:11:57] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:11:58] "GET / HTTP/1.1" 200 23623
[16/Oct/2023 10:12:32] "GET / HTTP/1.1" 200 23619
[16/Oct/2023 10:12:33] "GET / HTTP/1.1" 200 23619
/var/www/shoppingcart/index/views.py changed, reloading.
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:13:03
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:13:03] "GET / HTTP/1.1" 500 64265
Internal Server Error: /
Traceback (most recent call last):
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "/home/gulden/.local/lib/python3.10/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/var/www/shoppingcart/index/views.py", line 93, in index
    msg = request.session['msg']
  File "/home/gulden/.local/lib/python3.10/site-packages/django/contrib/sessions/backends/base.py", line 53, in __getitem__
    return self._session[key]
KeyError: 'msg'
[16/Oct/2023 10:13:04] "GET / HTTP/1.1" 500 64265
Watching for file changes with StatReloader
Performing system checks...

System check identified no issues (0 silenced).
October 16, 2023 - 10:20:52
Django version 4.2.6, using settings 'shoppingcart.settings'
Starting development server at http://10.21.17.68:8080/
Quit the server with CONTROL-C.

[16/Oct/2023 10:20:56] "GET / HTTP/1.1" 200 22795
[16/Oct/2023 10:20:59] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 10:20:59] "GET / HTTP/1.1" 200 24425
[16/Oct/2023 10:21:00] "GET /add/1f1321bb-0542-45d0-9601-2a3d007d5842 HTTP/1.1" 302 0
[16/Oct/2023 10:21:00] "GET / HTTP/1.1" 200 24664
[16/Oct/2023 10:21:02] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 10:21:02] "GET / HTTP/1.1" 200 23192
[16/Oct/2023 10:21:17] "GET / HTTP/1.1" 200 23032
[16/Oct/2023 10:21:52] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 10:21:52] "GET / HTTP/1.1" 200 24662
[16/Oct/2023 10:22:40] "GET / HTTP/1.1" 200 24605
[16/Oct/2023 10:22:49] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 10:22:49] "GET / HTTP/1.1" 200 24605
[16/Oct/2023 10:22:54] "GET /add/f9a6d214-1c38-47ab-a61c-c99a59438b12 HTTP/1.1" 302 0
[16/Oct/2023 10:22:54] "GET / HTTP/1.1" 200 24605
[16/Oct/2023 10:22:56] "GET /make-order/ HTTP/1.1" 302 0
[16/Oct/2023 10:22:56] "GET / HTTP/1.1" 200 23335
